protocols/imap/src/main/java/org/apache/james/imap/processor/AbstractMessageRangeProcessor.java

```diff
@@ -86,7 +86,7 @@ protected Mono<Void> processRequestReactive(R request, ImapSession session, Resp
                     getOperationName(), session.getSelected().getMailboxId(), targetMailbox, request.getIdSet(), e));
             })
             .onErrorResume(OverQuotaException.class, e -> {
-                taggedBad(request, responder, HumanReadableText.FAILURE_OVERQUOTA);
+                no(request, responder, HumanReadableText.FAILURE_OVERQUOTA);
```

Comment:
```suggestion
                no(request, responder, HumanReadableText.FAILURE_OVERQUOTA, StatusResponse.ResponseCode.overQuota());
```

Here and in other places.